lib/ReactViews/Custom/Chart/ChartPanelDownloadButton.tsx

```diff
@@ -69,8 +69,8 @@ async function download(items: TableMixin.Instance[]) {
   const synthesized = synthesizeNameAndValueArrays(items);
 
   // Could implement this using TaskProcessor, but requires webpack magic.
-  const worker = await import("worker-loader!./downloadHrefWorker").then(
-    (labelLayoutWorker) => new labelLayoutWorker.default()
+  const worker = new Worker(
```

Comment:
My interpretation is that this was the intention when migrating to webpack 5, but something didn't work, so the dependency on worker-loader was kept and will be removed in a separate PR.